Screenbox/Converters/ThicknessFilterConverter.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Converter ported to C#.
+// Source: https://github.com/microsoft/microsoft-ui-xaml/pull/6829
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Screenbox.Converters;
+
+/// <summary>
+/// Value converter that takes a <see cref="Thickness"/> value and returns a modified value.
+/// Can be used to convert a <see cref="Thickness"/>, side, or edge value to zero.
+/// </summary>
+public class ThicknessFilterConverter : IValueConverter
+{
+    public enum ThicknessFilterKind { None, Left, Top, Right, Bottom, LeftRight, TopBottom };
+
+    /// <summary>
+    /// Identifies the <see cref="FilterProperty"/> property.
+    /// </summary>
+    public static readonly DependencyProperty FilterProperty = DependencyProperty.Register(
+        nameof(Filter), typeof(ThicknessFilterKind), typeof(ThicknessFilterConverter), new PropertyMetadata(null));
+
+    public ThicknessFilterKind Filter { get; set; } = ThicknessFilterKind.None;
```

Comment:
I think the property name `Filter` is confusing. This converter converts from Thickness to Thickness with some edge zero out. The name makes me think it is filtering four values of Thickness into a single number. Maybe `Exclude` is a more descriptive name.

Reply:
To me it conveys the objective pretty clearly (filters out a value), and it aligns with the WinUI terminology (the [inspiration](https://github.com/marcelwgn/microsoft-ui-xaml/blob/dev/splitbutton-dark-theme-border-fix/dev/SplitButton/SplitButton.xaml) and the [corner radius](https://github.com/microsoft/microsoft-ui-xaml/blob/winui2/main/dev/CommonStyles/CornerRadius_themeresources.xaml) converter)